src/controllers/user.js

```diff
@@ -68,7 +68,7 @@ export const signin = (req, res) => {
           jwt.verify(token, process.env.JWT_KEY, (err, data) => {
            
           });
-          res.json({ success: true, token: `JWT ${token}` });
+          res.json({ success: true, token: `JWT ${token}`,user });
```

Comment:
Consider not returning password in the user info you are returning here.

Reply:
The password in user info is encrepted. Is there any virnerability?

Reply:
No matter how it's encrypted, it should not be exposed